scripts/osx/newscoop_osx_install.sh

```diff
@@ -85,6 +85,9 @@ chmod 775 plugins install cache images public conf log
 # default admin / password
 composer self-update
 composer install --prefer-dist
-sudo ./application/console newscoop:install --fix --database_name newscoop --database_user root --no-client
+sudo ./application/console newscoop:install --fix --database_name newscoop --database_user root --database_password root --no-client
 sudo php upgrade.php
 sudo php $SCRIPTPATH/../../newscoop/scripts/fixer.php
+
+# create an apache htaccess file after the installation process
+cp $SCRIPTPATH/htaccess.dist $SCRIPTPATH/.htaccess
```

Comment:
I'm working on installing on exotic systems at the moment. I'll revisit this and see if I can fix it properly in a bit.